src/signup.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import './App.css';
+
+ export function SignUp(){
+    return(
+        <div class="conatiner-bg bg-image">
+            <div class="mask">
```

Comment:
You are using the `class` keyword all over the place which is not allowed in JSX because it's a reserved JavaScript keyword. As mentioned before, the HTML's class attribute equivalent is JSX is the `className` prop.